src/crepe/api/Config.h

```diff
@@ -11,19 +11,18 @@ namespace crepe {
  * modified *before* execution is handed over from the game programmer to the engine (i.e. the
  * main loop is started).
  */
-class Config {
+class Config final {
```

Comment:
Dit is er toch voor om ervoor te zorgen dat je user deze klass niet mag overerfen.

Reply:
Klopt